Azure/azure-cli
# [Storage] `az storage account create/update`: Add --public-network-access parameter

**Description**

Add --public-network-access parameter to `az storage account create` and `az storage account update`. The allowed values are  'Enabled' and 'Disabled'

**Testing Guide**

Currently only euap regions have --public-network-access parameter enabled. 
`az storage account create -g {resource group name} -n {storage account name} --public-network-access Enabled`
The created storage account should have the publicNetworkAccess property set to "Enabled"
`az storage account update -g {resource group name} -n {storage account name} --public-network-access Disabled`
The storage account should now have the publicNetworkAccess property update to "Disabled"

---

This checklist is used to make sure that common guidelines for a pull request are followed.
